WordPress/Classes/Extensions/URL+Helpers.swift

```diff
@@ -2,6 +2,18 @@ import Foundation
 
 extension URL {
 
+    /// The URLResource fileSize of the file at the URLs, if available.
+    ///
+    /// - param URL: A file URL.
+    ///
+    var resourceFileSize: Int? {
```

Comment:
fileSize maybe will be a better name. 
And is Int large enough to hold the result? on objective C the fileSize use to be a `long long` type that normally converts to an `Int64`

The parameter documentation also needs to be updated. What is the result of it? bytes? Kbs? and we no longer need the param info.